app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_arguments = command.split(" ")
+    if len(splitted_arguments) != 3:
+        raise ValueError("Incorrect comand provided")
+    command, fname_to_copy, new_fname = command.split(" ")
```

Comment:
There is no need to perform an additional split, you could unpack the `splitted_arguments` variable.